docs/src/languages/php/fpm.md

```diff
@@ -1,103 +1,122 @@
 ---
 title: "PHP-FPM sizing"
+description: "Learn how to adjust the maximum number of PHP-FPM workers for your app"
 weight: 5
 ---
 
-Platform.sh uses a heuristic to automatically set the number of workers of the PHP-FPM runtime based on the memory available in the container. This heuristic is based on assumptions about the memory necessary on average to process a request. You can tweak those assumptions if your application will typically use considerably more or less memory.
+PHP-FPM helps improve your app's performance
+by maintaining pools of workers that can process PHP requests.
+This is particularly useful when your app needs to handle a high number of simultaneous requests. 
 
-Note that this value is independent of the `memory_limit` set in `php.ini`, which is the maximum amount of memory a single PHP process can use before it is automatically terminated.
-These estimates are used only for determining the number of PHP-FPM workers to start.
+By default, Platform.sh automatically sets a maximum number of PHP-FPM workers for your app. 
+This number is calculated based on three parameters:
+- The container memory: the amount of memory you can allot for PHP processing 
+  depending on [app size](../../create-apps/app-reference.md#sizes).
+- The request memory: the amount of memory an average PHP request is expected to require.
+- The reserved memory: the amount of memory you need to reserve for tasks that aren't related to requests. 
 
-## The heuristic
+The calculation is performed as follows: ![FPM](/images/php/PHP-FPM-Workers-Calculation.png "0.2")
```

Comment:
:hammer_and_wrench: I'm somewhat allergic to the phrasing `is performed` as it usually seems to me to involve extra words. One of the two suggestions might fit?

Also added meaningful alt text.
```suggestion
The number is calculated as follows: ![The sum of container memory minus reserved memory divided by request memory](/images/php/PHP-FPM-Workers-Calculation.png "0.2")
```

Reply:
```suggestion
This is calculated as follows: ![The sum of container memory minus reserved memory divided by request memory](/images/php/PHP-FPM-Workers-Calculation.png "0.2")
```

Reply:
Didn't mean to trigger you! 😁 But I get it!